historical_surveillance/forms.py

```diff
@@ -6,12 +6,6 @@ class AggregateEnrollmentForms(forms.ModelForm):
     class Meta:
         model = AggregateEnrollment
         fields = "__all__"
-        widgets = {
```

Comment:
I removed these widget fields as they were causing an issue running the ./manage.py script. Removing them didn't seem to be braking anything. @mildredfakoya and @guezandy can you confirm that removing them shouldn't affect anything? Django seems to be smart enough to generate the fields appropriately.